flink-table/flink-table-common/src/test/java/org/apache/flink/table/types/inference/InputTypeStrategiesTest.java

```diff
@@ -58,15 +42,16 @@
 import static org.apache.flink.table.types.inference.InputTypeStrategies.or;
 import static org.apache.flink.table.types.inference.InputTypeStrategies.sequence;
 import static org.apache.flink.table.types.inference.InputTypeStrategies.varyingSequence;
-import static org.apache.flink.util.CoreMatchers.containsCause;
-import static org.hamcrest.CoreMatchers.equalTo;
-import static org.junit.Assert.assertThat;
 
 /**
  * Tests for built-in {@link InputTypeStrategies}.
  */
 @RunWith(Parameterized.class)
-public class InputTypeStrategiesTest {
+public class InputTypeStrategiesTest extends InputTypeStrategiesTestBase {
+
+	public InputTypeStrategiesTest(TestSpec testSpec) {
```

Comment:
why is this required?

Reply:
As above. Did not know the `Parameter` can be in a separate class than `Parameters`. Therefore I was passing the parameters through constructor. As I moved the `RunWith` to the base class, it is no longer needed.